server_admin/topics/authentication/webauthn.adoc

```diff
@@ -17,38 +17,33 @@ The setup procedure of WebAuthn support for 2FA is the following :
 
 . Click *Authentication* in the menu.
 . Click the *Required Actions* tab.
-. Click *Register*.
-. Click the *Required Action* drop-down list.
-. Click *Webauthn Register*.
-. Click *Ok*.
+. Toggle the *Webauthn Register* switch to *ON*.
 
-Mark the *Default Action* checkbox if you want all new users to be required to register their WebAuthn credentials.
+Toggle the *Default Action* switch to *ON* if you want all new users to be required to register their WebAuthn credentials.
 
 [[_webauthn-authenticator-setup]]
 ===== Adding WebAuthn authentication to a browser flow
 
 . Click *Authentication* in the menu.
 . Click the *Browser* flow.
-. Click *Copy* to make a copy of the built-in *Browser* flow.
+. Select *Duplicate* from the "ellipses button" to make a copy of the built-in *Browser* flow.
```

Comment:
From the *Action* list, select *Duplicate*.

For more details on wording for GUI elements, see https://docs.engineering.redhat.com/display/CCSENABLE/GUI+elements